tracer/dependabot/Datadog.Dependabot.Vendors.csproj

```diff
@@ -27,11 +28,8 @@
     <!-- https://www.nuget.org/packages/dnlib/3.4.0 -->
     <PackageReference Include="dnlib" Version="3.4.0" />
 
-    <!-- https://www.nuget.org/packages/Datadog.Sketches/1.0.0 -->
-    <PackageReference Include="Datadog.Sketches" Version="1.0.0" />
-
-    <!-- https://www.nuget.org/packages/IndieSystem.Text.RegularExpressions/0.6 -->
-    <PackageReference Include="IndieSystem.Text.RegularExpressions" Version="0.6" />
+    <!-- https://www.nuget.org/packages/Datadog.Sketches/main -->
+    <PackageReference Include="Datadog.Sketches" Version="main" />
```

Comment:
`Version="main"`

Okay I think this one is on me
I don't any action here, but I may have messed up the code that generates the value for this specific instance